src/Platform.cpp

```diff
@@ -91,6 +117,16 @@ FilterEngine& Platform::GetFilterEngine()
   return *std::shared_future<FilterEnginePtr>(filterEngine).get();
 }
 
+Updater& Platform::GetUpdater()
+{
```

Comment:
I would like to have creation of `Updater` and `evaluatedJsSources` to be thread safe, but perhaps it's actually fine for present because I suspect that all these creations will be re-engineered soon anyway.

Reply:
Good point, it crossed my mind when I started writing this code. For the Updater thread safety, since Updater does not require async callback from JS code for creation, I propose to change constructor to:
```
Updater& Platform::GetUpdater()
{
  {
    std::lock_guard<std::mutex> lock(modulesMutex);
    if (updater)
        return *updater;
  }
  GetJsEngine(); // ensures that JsEngine is instantiated
  {
    std::lock_guard<std::mutex> lock(modulesMutex);
    if (!updater)
        updater = std::make_shared<Updater>(jsEngine, GetEvaluateCallback());
    return *updater;
  }
}
```
And I'll add a dedicated mutex to serialize access to evaluatedJsSources.